### ISSUE TITLE
Using Save with Filepath during Load doesn't work correctly

### ISSUE THREAD

**Issue by PrzemyslawKlys** [EvotecIT/OfficeIMO#12]
Currently, when you load document and try to save it as another document it seems to save as both `filePath31 and `filePath4`.

```csharp
            using (WordDocument document = WordDocument.Load(filePath3)) {
                Assert.True(filePath3.IsFileLocked() == true);

                Assert.True(document.Paragraphs.Count == 3);
                document.AddParagraph("More paragraphs!");
                Assert.True(document.Paragraphs.Count == 4);
                document.Save(filePath4);
            }
```

This needs to be fixed, but I am not sure how, as my tries failed.

**Comment by SiL3NC3** [comment]
Yes, even with calling the Load function with the parameters "readonly = true" and "autosave = false", sadly.

**Comment by PrzemyslawKlys** [comment]
Yes, I'm aware and it's a bit weird because I actually pass this to OpenXMLSDK I believe and it's still failing.

**Comment by PrzemyslawKlys** [comment]
Fixed with #16
